### ISSUE TITLE
Failure in LSU region extraction due to S4 class operator error

### ISSUE THREAD

**Issue by vmikk** [brendanf/LSUx#6]
Hello Brendan,

I'm encountering an issue when trying to run LSUx. It fails during the LSU region extraction step, producing the error:
`Error in aln$alignment : $ operator not defined for this S4 class`.

The functions `inferrnal::cmsearch` and `inferrnal::cmalign` are functioning correctly, but `extract_LSU` fails.
I believe the error occurs at this specific line of code:
https://github.com/brendanf/LSUx/blob/1423e2d42e369e1c8682ac2abeb0f75ff0d2cc19/R/lsux.R#L471

It seems the function might need to be invoked differently, perhaps it should be called using:
```
LSUx:::extract_LSU.character(aln = as.character(aln@unmasked), rf = aln@GC$RF)
```

To illustrate the issue, here is a minimal reproducible example that triggers the error:
```
library(LSUx)

seq <- system.file("extdata/sample.fasta", package = "inferrnal")

cm_32S_trunc <- system.file(
    file.path("extdata", "fungi_32S_LR5.cm"),
    package = "LSUx")

regions <- lsux(seq = seq, cm_32S = cm_32S_trunc, ITS1 = TRUE, cpu = 1, glocal = TRUE,  quiet = FALSE)
```

This was tested on a Linux system using the current development version of `LSUx` (v0.99.6) and `inferrnal` (v0.99.7), both installed from GitHub.

**Comment by brendanf** [comment]
Thanks for the report Vladimir.  This is due to changes in the interface for `inferrnal` in version 0.99.7, and it should be corrected now in 51de0bcd.  Let me know if you run in to any more issues.

**Comment by vmikk** [comment]
Great, now it works! Thank you!
